ingest/src/server_lora.rs

```diff
@@ -97,13 +103,18 @@ impl poc_lora::PocLora for GrpcServer {
             .map_err(|_| Status::invalid_argument("invalid signature"))?;
 
         let event_id = EventId::from(&event);
-        match file_sink::write(&self.lora_witness_report_tx, report).await {
-            Ok(_) => tracing::debug!(
+        let _ = file_sink::write(
+            "ingest_server_lora_witness_report",
+            &self.lora_witness_report_tx,
+            report,
+        )
+        .await
+        .map(|_| {
+            tracing::debug!(
                 "successfully processed witness report, returning response {}",
```

Comment:
Since the `Err(...)` clause of the original `match` went away, the debug log message shouldn't assume success:
```suggestion
                "processed witness report, returning response {}",
```

Reply:
@dpezely `.map` maps `Ok` case and `.map_err` map the `Err` one and both cases are already logged generically within `write`.

So this one is kinda redundant, the only reason I put it back is because I saw that it was logging `event_id` on success (which can't be replicated generically by `write`) and thought that maybe this particular message is operationally relied upon - otherwise I'd rather remove this logging map altogether.

@bbalser do you know if this log message is particularly relied upon by anyone?

Reply:
> To my knowledge our logs aren't going anywhere, so I don't believe this log is being relied upon.

Right, but people could also be looking for this message manually.

Reply:
I resolved this by adding an optional logging context parameter, where things like `event_id` can be passed and subsequently included in the otherwise-generic log statements: https://github.com/helium/oracles/pull/94/commits/88d40d4ed408b6eefa88a700d3223a8f7ea6856e